Updates/Bug Fixes

tikimonarch opened the pull request:
- add -> full info
- swappable, optional add command
- swappable info command
- catch index error for delete

tikimonarch committed:
fixed bugs for data, calculator

tikimonarch committed:
Major Issue Fixes

- add -> full information
- swappable, optional add command
- swappable info command
- catch index error for delete

tikimonarch committed:
Merge branch 'master' into MainBranch

HengFuYuen commented on src/main/java/seedu/duke/Parser.java:
```diff
@@ -102,30 +132,57 @@ private static String getProcessedAdd(String userInput, FoodList foodList) throw
+            case "g/":
+                String processGender = trimmedParam;
+                if (processGender.equals("M")) {
+                    gender = Gender.MALE;
```
Perhaps consider changing female to an if-else case and the else case can throw an exception. Maybe you would like to consider this for activity level as well.

HengFuYuen commented on src/main/java/META-INF/MANIFEST.MF:
```diff
@@ -0,0 +1,3 @@
+Manifest-Version: 1.0
+Main-Class: seedu.duke.DietBook
```
I am not very sure if an update would be good/necessary for this file but perhaps you can consider.

HengFuYuen reviewed (approved):
Perhaps some changes would reduce bugs.

Current commit LGTM.

tikimonarch committed:
Improved input parser

tikimonarch committed:
Merge branch 'master' into MainBranch

tikimonarch committed:
Major OOP Update

- created the entire command class -> 11 commands
- shift parse -> manager (renamed to manage)
- removed divider line from main
- main execution

tikimonarch committed:
Input Checker

-Implemented input checker class

HengFuYuen commented on src/main/java/seedu/dietbook/Manager.java:
```diff
@@ -1,15 +1,24 @@
 package seedu.dietbook;
 
+import seedu.dietbook.command.*;
```
Perhaps you might want to make a specific imports rather than wildcard imports.

tikimonarch replied:
Understandable but the reason for all is because all command is used so there would be like 11 import statements if i am specific? what do you think?

HengFuYuen reviewed (approved):
Perhaps you can consider leaving an empty line between description and parameter section for JavaDocs comment and remove the wildcard imports.

Current commit LGTM.

tikimonarch committed:
New check, bug fixes

->fixed for acticity, gender, calculate type checks
->new check if an option is specified but with an empty field

tikimonarch committed:
Spacings
